mbf_utility/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.0.2)
 project(mbf_utility)
 
 find_package(catkin REQUIRED COMPONENTS
+  base_local_planner
```

Comment:
Didn't we had some issues with this dependency? Or why did we copy the footprint helper?

Reply:
yup.....  https://github.com/magazino/move_base_flex/issues/184